drivers/homeless_summary_report/app/views/homeless_summary_report/warehouse_reports/reports/_form.haml

```diff
@@ -1,16 +1,46 @@
-%p.alert.alert-info Please note that, unless you include all homeless project types (ES, SO, SH, TH), returns to homelessness calculations may not work as expected.
 - url = polymorphic_path(@report.report_path_array)
-= simple_form_for @filter, as: :filters, url: url do |f|
+
+%p.alert.alert-info Please note that, unless you include all homeless project types (ES, SO, SH, TH), returns to homelessness calculations may not work as expected.
+
+= simple_form_for @filter, as: :filters, url: url, data: { controller: 'filter-projects' } do |f|
   - content_for :filters_col_full do
-    .row
+    .row.mb-4
+      .col-sm-3
+        = f.input :start, as: :date_picker, label: 'Start Date', required: true
+      .col-sm-3
+        = f.input :end, as: :date_picker, label: 'End Date', required: true
+      .col-sm-6
+        = f.input :coc_codes, collection: @filter.coc_code_options_for_select(user: current_user), as: :select_two, input_html: {multiple: true, placeholder: 'Choose CoCs'}, label: 'CoC Codes'
+    %h2 Projects to Include
+    %p.alert.alert-info Please note, the following options are additive.  If you choose a single project and a project type, the report will run for all projects in the project type and the chosen project.  If you choose a data source and project type, the report will run for all projects in the chosen data source and chosen project type.
+    .row.mb-4
+      .col-md-8
+        .row
+          .col-sm-6
+            = f.input :project_ids, collection: @filter.project_options_for_select(user: current_user), as: :grouped_select_two, group_method: :last, input_html: { multiple: true, placeholder: 'Choose Projects', data: { 'filter-projects-target' => 'projects', action: 'change->filter-projects#update' } },label: 'Projects'
+          .col-sm-6
+            = f.input :data_source_ids, collection: @filter.data_source_options_for_select(user: current_user), as: :select_two, input_html: {multiple: true, placeholder: 'Choose Data Sources', data: { 'filter-projects-target' => 'dataSources', action: 'change->filter-projects#update' }}, label: 'Data Sources'
+        .row
+          .col-sm-6
+            = f.input :project_type_codes, collection: @filter.available_residential_project_types, input_html: { multiple: true, placeholder: 'Choose Project Types', data: { 'filter-projects-target' => 'projectTypes', action: 'change->filter-projects#update' } }, label: 'Project Types', as: :select_two
```

Comment:
I changed this from `GrdaWarehouse::Hud::Project::PROJECT_GROUP_TITLES.invert` which I had there before. Figured it should match the SPM filter. https://github.com/greenriver/hmis-warehouse/pull/2162/commits/714a1121c94f02d4081519fb480bc59ffbc3397c